src/app/components/LinkedData/index.tsx

```diff
@@ -129,16 +151,10 @@ const LinkedData = ({
     },
     ...(isTrustProjectParticipant && { noBylinesPolicy }),
   };
-
-  const bylineAuthor = {
-    '@type': 'Person',
-    name: authorName,
-    ...(sameAs.length && { sameAs }),
-    ...(authorImage && { image: authorImage }),
-  };
-
-  const author = hasByline ? bylineAuthor : orgAuthor;
-
+  let author;
+  if (hasByline && bylineAuthors) {
+    author = bylineAuthors.length === 1 ? bylineAuthors?.[0] : bylineAuthors;
+  } else author = orgAuthor;
```

Comment:
This is a wee bit easier to read (+ spacing)
```suggestion
  
  let author = orgAuthor;
  if (hasByline && bylineAuthors) {
    author = bylineAuthors.length === 1 ? bylineAuthors?.[0] : bylineAuthors;
  }
```